azure/inputsources/shared/masterdata.ftl

```diff
@@ -1142,13 +1142,22 @@
             }
           }
         },
+        "docker-debian" : {
+          "Index" : 16,
+          "ProtectedSettings" : {
+            "exec" : {
+              "Key" : "commandToExecute",
+              "Value" : "sudo apt install docker.io -y'"
```

Comment:
Does this need to include apt-get update to refresh the apt dB?

Reply:
That's performed by the `update-debian` boot strap, so it doesn't have to run a zillion times.